rpcd/playbooks/rpc-pre-upgrades.yml

```diff
@@ -18,3 +18,17 @@
   user: root
   roles:
     - { role: "rpc_pre_upgrade", tags: [ "rpc-pre-upgrades" ] }
+
+- name: restart beaver
+  hosts: all
+  tasks:
```

Comment:
@mancdaz To make it simpler, you could have ` tags: "rpc-pre-upgrades"` set at the play level instead of task level, which avoids repetition.